var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -22,9 +22,9 @@ class Ginkgo(CMakePackage, CudaPackage, ROCmPackage):
 
     version("develop", branch="develop")
     version("master", branch="master")
-    version("1.5.0.glu", branch="glu")
+    version("1.5.0", branch="develop", preferred=True)  # v1.5.0
```

Comment:
Surely this should have its own commit or tag?

Reply:
Indeed, I was in the process of releasing our package when you made this comment. I wanted to make sure everything is fine w.r.t. the other packages first, so that we don't need a tiny bugfix release soon after the main one...